db-test.sql

```diff
@@ -0,0 +1,118 @@
+-- MySQL dump 10.19  Distrib 10.3.28-MariaDB, for Linux (x86_64)
```

Comment:
Why do we need `db-test.sql`? If the user wants to have a database for testing and a database for production, they can just set them up themselves using the same file for both (with some minor edits). This is slightly counter-intuitive given that we don't use this database for the test workflow.

Reply:
We don't.  I had originally provided this to separate production from development, but we can rely on users to do this form themselves.  I will remove this file from the repo.